README.md

```diff
@@ -1,15 +1,6 @@
-# Super cool project name
+# Using Unsupervsed Learned to provide a Competive Edge in a Classification Competition
```

Comment:
- some typos here: "Using unsupervised learning", and "competitive"
- i am not so sure what does "a competitive edge" mean?